site/src/examples/static-list/DefaultMultiple.tsx

```diff
@@ -0,0 +1,26 @@
+import { Label, StackLayout, Text } from "@salt-ds/core";
```

Comment:
Remove Default from example naming... could just be Multiple

Reply:
removed